static/js/initCardMapView.js

```diff
@@ -121,4 +53,94 @@ function clearMarkers() {
         markers[i].setMap(null);
     }
     markers = [];
-}
\ No newline at end of file
+}
+
+
+if (navigator.geolocation) {
+    console.log('Geolocation is supported!');
+}
+else {
+    console.log('Geolocation is not supported for this Browser/OS.');
+}
+
+window.onload = function () {
+    var startPos;
+    var geoSuccess = function (position) {
+        startPos = position;
+
+        var token = $("#token").val();
+
+        urlAPI = "http://54.241.216.252:5000/mappedcafes/{0}/{1}/{2}"
+            .format(
+                token,
+                startPos.coords.latitude,
+                startPos.coords.longitude
+            );
+
+        tmpUrl = "http://54.241.216.252:5002/mappedcafes/4235c90663f34d6bb90d4e2c8e2bf875";
+
+        console.log(urlAPI);
```

Comment:
is this part of production code?

Reply:
Yes. tmpUrl is test url. When API development is completed, I intend to change it.